.github/workflows/ci_workflow.yml

```diff
@@ -6,31 +6,6 @@ name: Test tap-facebook
 on: [push]
 
 jobs:
-  linting:
```

Comment:
@edgarrmondragon is there are particular reason for removing the linting workflow?

Reply:
@pnadolny13 Yeah, because all the checks are now performed by pre-commit